samtranslator/schema/aws_serverless_connector.py

```diff
@@ -48,4 +53,5 @@ class EmbeddedConnector(BaseModel):
     DependsOn: Optional[PassThroughProp]
     DeletionPolicy: Optional[PassThroughProp]
     Metadata: Optional[PassThroughProp]
-    UpdatePolicy: Optional[PassThroughProp]
```

Comment:
Forgive me if this is a dumb question, but could renaming/removing this break any existing templates?

Reply:
It shouldn't since its the schema